tests_e2e/lisa/testsuites/agent-bvt.py

```diff
@@ -0,0 +1,41 @@
+from assertpy import assert_that
+from pathlib import Path
+from tests_e2e.lisa.tests.agent_bvt import custom_script
+
+from lisa import (
+    CustomScriptBuilder,
+    Logger,
+    Node,
+    simple_requirement,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+)
+from lisa.sut_orchestrator.azure.common import get_node_context
+
+
+@TestSuiteMetadata(
+    area="bvt",
+    category="functional",
+    description="""
+    A POC test suite for the waagent BVTs.
+    """,
+    requirement=simple_requirement(unsupported_os=[]),
+)
+class AgentBvt(TestSuite):
+    @TestCaseMetadata(description="", priority=0)
+    def check_agent_version(self, node: Node, log: Logger) -> None:
+        script_path = CustomScriptBuilder(Path(__file__).parent.parent.joinpath("tests", "agent_bvt"), ["check_agent_version.py"])
+        script = node.tools[script_path]
+        result = script.run()
+        log.info(result.stdout)
+        log.error(result.stderr)
+        assert_that(result.exit_code).is_equal_to(0)
+
+    @TestCaseMetadata(description="", priority=0)
+    def custom_script(self, node: Node) -> None:
+        node_context = get_node_context(node)
+        subscription_id = node.features._platform.subscription_id
```

Comment:
Currently reaching into the internal "_platform" object to get the Subscription ID. This should be done by implementing a LISA feature that exposes subscription ID/resource group name/vm name.

Reply:
Once we are creating multiple test VMs with different images, how will LISA know which nodes this test should be ran with? Will that be specified in the runbook?

Reply:
Yes, we will create a combinator to execute each test on each distro and add it to the runbook

Reply:
yes, the Node represents the VM created by LISA